src/django_idom/config.py

```diff
@@ -15,7 +15,7 @@
 _CACHES = getattr(settings, "CACHES", {})
 if _CACHES:
     if "idom_web_modules" in getattr(settings, "CACHES", {}):
-        IDOM_WEB_MODULE_CACHE = "idom_web_modules"
+        IDOM_WEB_MODULE_CACHE = "idom"
```

Comment:
Should we rename `IDOM_WEB_MODULE_CACHE` to `IDOM_CACHE` as well?

Reply:
Looks like I already handled this in PR #50 so I'm going to close this PR.